check-events.js

```diff
@@ -4,29 +4,17 @@
 
 const fs = require('fs')
 const path = require('path')
+const zlib = require('zlib')
 
 const TIME_FORMAT = /(201[78])-([0-9]{2})-([0-9]{2})-([0-9]{2})-([0-9]{2})/
-const CATEGORY_FORMAT = /^logging\.s3\.fxa\.([a-z]+)_server/
+const CATEGORY_FORMAT = /logging\.s3\.fxa\.([a-z]+)_server/
 const VERBOSE = false
 
-const args = process.argv
```

Comment:
Basically you can pass an arbitrary number of directories on the command-line and it'll read them all (only to one level deep)